tables/mdm/mdm.go

```diff
@@ -65,22 +77,17 @@ func MDMInfoColumns() []table.ColumnDefinition {
 }
 
 func MDMInfoGenerate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
-	profiles, err := getMDMProfile()
-	if err != nil {
-		return nil, err
-	}
+	profiles, _ := getMDMProfile()
 
-	depEnrolled, depCapable, userApproved := "unknown", "unknown", "unknown"
+	depEnrolled, userApproved := "unknown", "unknown"
 	status, err := getMDMProfileStatus()
 	if err == nil { // only supported on 10.13.4+
 		depEnrolled = strconv.FormatBool(status.DEPEnrolled)
 		userApproved = strconv.FormatBool(status.UserApproved)
 	}
 
-	depstatus, err := getDEPStatus()
-	if err == nil {
-		depCapable = strconv.FormatBool(depstatus.DEPCapable)
-	}
+	depstatus, _ := getDEPStatus(status)
```

Comment:
Same comment as above. Please leave a code comment explaining why its safe to throw out the error